test/helpers/position_assertions.cc

```diff
@@ -1472,6 +1473,12 @@ void checkAllForQuery(std::string query, const vector<shared_ptr<SymbolSearchAss
         for (auto &symbolInfo : *symbolInfos) {
             symbolAssertionMatches.emplace_back(*symbolInfo, nullptr);
         }
+        constexpr int MAX_RESPONSES = 50;
+        if (symbolInfos->size() > MAX_RESPONSES) {
+            ADD_FAILURE() << fmt::format(
+                "Too many responses for a `workspace/symbol` request. [Expected no more than {}, got {}].",
```

Comment:
Nit: Can you include info on which request failed?

Reply:
Fixed.  Now it looks like this:
```
test/helpers/position_assertions.cc:1478: Failure
Failed
Too many results for a `workspace/symbol` request for `hash`. [Expected no more than 50, got 51].
```